### PR TITLE
Updating version to 4.0.0 and adding changelog

### PR DESCRIPTION
**Pull request by nemakam**
This is for Azure Service Bus SDK 4.0.0

### CODE REVIEW

**Comment by nemakam** [comment]
@jsquire @weshaggard - Where do I add reference to the changelog.md file?

File: sdk/servicebus/Microsoft.Azure.ServiceBus/changelog.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# 4.0.0
+## Breaking Change
```
Comment (jsquire) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/changelog.md]: nit:  This should probably be "Breaking Changes" since it's multiple items in a list.

**Review by jsquire — APPROVED** [review]

**Comment by jsquire** [comment]
> @jsquire @weshaggard - Where do I add reference to the changelog.md file?

Not sure that I understand the question.   Could you offer a bit more context about what you're trying to do?

**Comment by nemakam** [comment]
> Not sure that I understand the question. Could you offer a bit more context about what you're trying to do?

I took a look at https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/releases.md#changelog-guidance. According to that, changelog should be part of nuspec, and I am not quite sure how to do that. Essentially, I see that the engineering pipeline automates the process of reading the changelog and creating a Github release notes based off that. But in order to achieve that, the link to `changelog.md` should be specified in `.nupkg`. Is that right?

**Comment by jsquire** [comment]
> > Not sure that I understand the question. Could you offer a bit more context about what you're trying to do?
> 
> I took a look at https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/releases.md#changelog-guidance. According to that, changelog should be part of nuspec, and I am not quite sure how to do that. Essentially, I see that the engineering pipeline automates the process of reading the changelog and creating a Github release notes based off that. But in order to achieve that, the link to `changelog.md` should be specified in `.nupkg`. Is that right?

On, that.... I'm not entirely sure.   I'm going to defer to @weshaggard, @danieljurek, and @mitchdenny as the experts in that area.

**Comment by weshaggard** [comment]
@scbedd correct me if I'm wrong but I don't believe you need to include the changelog in the nuget package itself we simply pull the data from it when we create the github releases, correct?

**Comment by scbedd** [comment]
@weshaggard That's correct, I just need to be able to pull it from _somewhere_. 

That I recall, we did not load the changelog content into the nuspec, if we _did_ do that the script would have an easy time pulling it yes.

Conversely, if that wasn't possible, we'd need to add the `changelog.md` to the nupkg so that the release script could pull _that_ out and parse it.

Either way works, just need one of them to be available.

**Comment by weshaggard** [comment]
@scbedd I guess I'm still a little confused. While I get we didn't include this information in the nuspec it does appear as though we need to included in the nupkg package itself in order for this to work. Which as best I can tell no nupkg package is currently doing. So we probably need to provide some more specific guidance on how to do that. @chidozieononiwu could you take a look and write up some guidance, or better yet figure out a way to automatically include the changelog.md in the common targets so that it gets included in our packages?

**Comment by weshaggard** [comment]
@chidozieononiwu we already have issue https://github.com/Azure/azure-sdk-for-net/issues/5824 tracking that which I've assigned to you.

@nemakam for this PR just having the changelog should be enough we will update our infra to properly include it.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**